tests/console/zypper_lr.pm

```diff
@@ -16,6 +16,8 @@ use testapi;
 use utils;
 
 sub run() {
+    # ZYPPER_LR is needed for inconsistent migration, test would fail looking for deactivated addon
+    set_var 'ZYPPER_LR';
```

Comment:
MIGRATION_INCONSISTENCY_DEACTIVATE will remove/deactivate add-on, that change will affect following test zypper_lr, before this change is test repos_check, so I created variable ZYPPER_LR to identify test zypper_lr.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L972
Other option could be alter variable SCC_ADDONS=we,sdk and remove sdk. What is clearer approach or do you have other options ?